src/modules/ContentList/index.tsx

```diff
@@ -123,8 +133,49 @@ const mockData = [
     },
 ];
 
+interface ContentListSharedProps {}
+
+/** @notExported */
+export interface ContentListUIProps extends ContentListSharedProps {
+    company?: CompanyFilterResult;
+    content?: ContentListContent;
+    onBackFromContent?: MouseEventHandler;
+    onChangeSearch?: ChangeHandler<string>;
+    onSelectCompany?: ChangeHandler<CompanyFilterResult>;
+    onSelectContent?: ChangeHandler<ContentListContent>;
+    onSelectContentType?: ChangeHandler<ContentType>;
+    searchTerm?: string;
+    selectedContentType: ContentType;
+}
+
 export function ContentListUI(props: ContentListUIProps): ReactElement {
-    const { company, onChangeSearch, onSelectCompany, onSelectContentType, searchTerm, selectedContentType } = props;
+    const {
+        company,
+        content,
+        onBackFromContent,
+        onChangeSearch,
+        onSelectCompany,
+        onSelectContent,
+        onSelectContentType,
+        searchTerm,
+        selectedContentType,
+    } = props;
+
+    if (content) {
+        return (
+            <Content
```

Comment:
For events we have the Transcript component load the transcripts and other event data it needs. I think we should probably do the same here since we don't want to query the content body for every article in the list, only the ones that are opened.